charts/openfga/values.yaml

```diff
@@ -161,3 +161,15 @@ nodeSelector: {}
 tolerations: []
 
 affinity: {}
+
+# @param sidecars Add additional sidecar containers to the pods
+# Example:
+# sidecars:
+#   - name: your-image-name
+#     image: your-image
+#     imagePullPolicy: Always
+#     ports:
+#       - name: portname
+#         containerPort: 1234
+sidecars: []
```

Comment:
Please add to `values.schema.json` as well.

Reply:
added